pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -1121,7 +1121,7 @@ func defaultPod() *api.Pod {
 		},
 		Spec: api.PodSpec{
 			SecurityContext: &api.PodSecurityContext{
-				// fill in for test cases
+			// fill in for test cases
```

Comment:
gofmt complains on this change (and one below).

Run `./hack/update-gofmt.sh` to make PR green.